src/chocolatey/infrastructure.app/commands/ChocolateyUpgradeCommand.cs

```diff
@@ -23,6 +23,7 @@ namespace chocolatey.infrastructure.app.commands
     using commandline;
     using configuration;
     using infrastructure.commands;
+    using licensing;

```

Comment:
I don't think that this is needed, and can be removed.

Reply:
I was going to tell you Visual Studio added it (which is true), but I took out the `License.ValidateLicense()` calls that needed it, so you're absolutely right this is not needed. I've rebased away the commit that added this.